### PR TITLE
Fix mypy aws example dags

### PR DESCRIPTION
**Pull request by subkanthi**
Fixes mypy errors in aws/example_dags

related: #19891 

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/main/UPDATING.md).

### CODE REVIEW

File: airflow/providers/amazon/aws/operators/eks.pyi [hunk 1]
```diff
@@ -0,0 +1,97 @@
+from typing import Optional, Dict, List
```
Comment (khalidmammadov) [inline airflow/providers/amazon/aws/operators/eks.pyi:18]: No License info needed?

Comment (potiuk) [reply airflow/providers/amazon/aws/operators/eks.pyi:18]: Rebasing to latest main and running pre-commit should add licence automatically @subkanthi

Comment (subkanthi) [reply airflow/providers/amazon/aws/operators/eks.pyi:18]: Done, thanks.

**Comment by potiuk** [comment]
Now pre-commits - to add the missing licences :)

**Comment by subkanthi** [comment]
> Now pre-commits - to add the missing licences :)

Thanks , added now.

**Review by potiuk — APPROVED** [review]

**Pull request merged**